composer.json

```diff
@@ -1,18 +1,15 @@
 {
   "name": "pluggit/monitoring",
   "description": "Monitoring abstraction layer",
+
   "require": {
-    "php": "^5.5|^7.0",
-    "psr/log": "^1.0"
+    "php": "^8.1",
+    "psr/log": "^3.0"
   },
   "require-dev": {
-    "phpspec/phpspec": "^2.5",
```

Comment:
These dependencies are already required by the phpspec/phpspec, we don't use them directly in the project